### PR TITLE
Adding the issue Templates

### LINKED ISSUE #22: Generic issue templates
Use LinkFree, for refrence.

### PR DESCRIPTION
**Pull request by Saurabhpatil-dev**
Hello, I am creating this PR regarding the issue resolve #22 .

### CODE REVIEW

File: ISSUE_TEMPLATE/config.yml
Comment (krishguptadev) [inline ISSUE_TEMPLATE/config.yml:4]:
```suggestion
  url: http://discord.eddiehub.org
```

**Review by krishguptadev — COMMENTED** [review]
I dun't think there should be a trailing `/`

File: ISSUE_TEMPLATE/config.yml
Comment (Saurabhpatil-dev) [reply ISSUE_TEMPLATE/config.yml:4]: Okay, I will make the change.

Comment (Saurabhpatil-dev) [reply ISSUE_TEMPLATE/config.yml:4]: @krishguptadev Then I guess it is an issue in Link Free also. Should I make PR there also?

**Comment by Saurabhpatil-dev** [comment]
@krishguptadev  Is it okay now ?

**Review by krishguptadev — APPROVED** [review]

File: ISSUE_TEMPLATE/bug.yml
Comment (Panquesito7) [inline ISSUE_TEMPLATE/bug.yml:3]: These labels do not exist on this repository. 👀
```suggestion
labels: ["bug"]
```

File: ISSUE_TEMPLATE/config.yml
Comment (Panquesito7) [inline ISSUE_TEMPLATE/config.yml:5]:
```suggestion
  about: Feel free to ask any questions on our Discord channel.
```

File: ISSUE_TEMPLATE/docs.yml
Comment (Panquesito7) [inline ISSUE_TEMPLATE/docs.yml:4]:
```suggestion
labels: ["documentation"]
```

File: ISSUE_TEMPLATE/feature_request.yml
Comment (Panquesito7) [inline ISSUE_TEMPLATE/feature_request.yml:4]:
```suggestion
labels: ["enhancement"]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: ISSUE_TEMPLATE/bug.yml
Comment (Saurabhpatil-dev) [reply ISSUE_TEMPLATE/bug.yml:3]: Okay.

Comment (krishguptadev) [reply ISSUE_TEMPLATE/bug.yml]: @Panquesito7 these are fallback files for other repos 🙂

**Review by Panquesito7 — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by CodingSpecies — COMMENTED** [review]
Is this pr good to go @Panquesito7 ? This has been open for a while, but has been approved. Unless you are waiting for a maintainer approval? Thank you.

**Comment by Panquesito7** [comment]
> Is this pr good to go @Panquesito7 ? This has been open for a while, but has been approved. Unless you are waiting for a maintainer approval? Thank you.


Hey there! For me, this looks good, but I can't merge it 😅 We'll have to wait on Eddie or someone who has merge permissions here.

**Comment by KendallDoesCoding** [comment]
Even thought this is merged, it doesn't seem to be added in the issues panel.

**Pull request merged**
